testGeogridOptical.py

```diff
@@ -225,18 +219,36 @@ def runGeogrid(info, info1, dem, dhdx, dhdy, vx, vy, srx, sry, csminx, csminy, c
 
     obj.runGeogrid()
 
-
-if __name__ == '__main__':
+    run_info = {
+        'chipsizex0': obj.chipSizeX0,
+        'vxname': vx,
+        'vyname': vy,
+        'sxname': srx,
```

Comment:
this should be `'sxname': kwargs.get('dhdxs')`

Reply:
Good catch! Done.